### DISCUSSION TITLE
Any more resources on learning this?  (youtube, discord, ...)

### DISCUSSION THREAD

**Discussion by punkouter26** [dotnet/eShop#164] [Q&A]
I would love to see it demoed and deployed to azure from scratch so I can figure out what I am doing wrong. I am currently looking for more resources to explain things better.

**Comment by captainsafia** [comment]
You may find the eShop on Azure sample helpful for this.

You can find it at https://github.com/Azure-Samples/eShopOnAzure/

**Comment by jflaga** [comment]
Hi @captainsafia , may I ask what the difference is between `dotnet/eShop` and `Azure-Samples/eShopOnAzure`?

**Comment by davidfowl** [comment]
The difference is the azure version uses azure resources 

- Identity server is replaced with Entra
- RabbitMQ is replaced by ServiceBus
- AppInsights is enabled

**Comment by jflaga** [comment]
Thanks @davidfowl

**Comment by jamesmontemagno** [comment]
Converting to discussion for future discussions

**Comment by Samadmehmood** [comment]
Please anyone initiate a slack workspace or discord channel for this? i can do one but the pioneer should be the admin.
As a .net developer i see a lot of potential in this project, i am used to nopcommerce but its frontend lacks up to par performance and here eShop could do a lot better, i am also interested to contribute along with my fellows but for that and conversation we must have a platform.
Thanks.

**Comment by cedrox** [comment]
Hello, is it possible to have a proper step by step to have the solution up and running both for netcore and Azure version ? 
On azure, there is quite few things to do to have the Entra configured... I am working on a script right now but ideally, it would be better to have terraform taking care of that...
Thanks
